test/unit/CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ endif()  # USE_EXTERNAL_GTEST
 # Build PROJ unit tests
 #
 
-if("${CMAKE_C_COMPILER_ID}" STREQUAL "MSVC" AND BUILD_LIBPROJ_SHARED)
+if(MSVC AND BUILD_SHARED_LIBS)
   add_definitions(-DPROJ_MSVC_DLL_IMPORT=1)
```

Comment:
It doesn't look like any tests use the `PROJ_MSVC_DLL_IMPORT` define, so I might remove this block.

Reply:
Since this PR is about renaming, the clean-up could be a separate PR.

Reply:
Hmm right, I'll keep this block for now. Restored.